fail2ban/server/filter.py

```diff
@@ -161,6 +163,43 @@ def getIgnoreRegex(self):
 		return ignoreRegex
 
 	##
+	# Add a regular expression which matches the reset.
+	#
+	# The regular expression can also match any other pattern than reets
+	# and thus can be used for many purporse.
+	# @param value the regular expression
+
+	def addResetRegex(self, value):
+		try:
+			regex = ResetRegex(value)
+			self.__resetRegex.append(regex)
+			if "\n" in regex.getRegex() and not self.getMaxLines() > 1:
+				logSys.warning(
+					"Mutliline regex set for jail '%s' "
```

Comment:
Shall I fix it in addFailRegex as well? - I just copied the exact text but didnt spot that.